qprofiler2/src/org/qcmg/qprofiler2/fastq/FastqSummaryReport.java

```diff
@@ -107,7 +107,7 @@ public void toXml(Element parent1) {
 	 * 
 	 * @return next row in file
 	 */
-	public void parseRecord(FastqRecord record) {
+	public void parseRecord(FastqRecord record) throws IllegalArgumentException {
```

Comment:
IllegalArgumentException is an unchecked exception and so you don't need it in the throws clause